website/docs/r/external_network.html.markdown

```diff
@@ -13,6 +13,8 @@ Requires system administrator privileges.
 
 Supported in provider *v2.2+*
 
+~> **Note:** For NSX-T suported external network please use [vcd_external_network_v2](/docs/providers/vcd/r/external_network_v2.html)
```

Comment:
I would say "and VCD 10+" as it have fixes and suport NSX-V

Reply:
It depends on what we state with `vcd_external_network`. It is still the only way for 9.7.